src/commands/test.ts

```diff
@@ -42,6 +42,12 @@ export async function testSolution(uri?: vscode.Uri): Promise<void> {
                 detail: "Test with the written cases in file",
                 value: ":file",
             },
+            {
+                label: "$(debug-restart) Re-run last...",
```

Comment:
One suggestion here: We should only show the re-run option when the `lastTest` is not empty. Also, this option should put at the top of the list, to make it more convenient for the users.